### PR TITLE
Update resource-usage-monitoring.md

### PR DESCRIPTION
**Pull request by njain1985**
Add New Relic Kubernetes monitoring integration

>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Please delete this note before submitting the pull request.
> For 1.12 Features: set Milestone to 1.12 and Base Branch to release-1.12
> Help editing and submitting pull requests:  https://deploy-preview-9510--kubernetes-io-master-staging.netlify.com/docs/contribute/start/#submit-a-pull-request.
> Help choosing which branch to use, see
> https://kubernetes.io/docs/contribute/start#choose-which-git-branch-to-use.
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>

### CODE REVIEW

**Comment by njain1985** [comment]
CNCF-CLA just signed, please approve this.

File: content/en/docs/tasks/debug-application-cluster/resource-usage-monitoring.md [hunk 1]
```diff
@@ -96,3 +96,13 @@ With [Dynatrace Kubernetes monitoring](https://www.dynatrace.com/technologies/cl
 Dynatrace automatically discovers all containers running on Kubernetes and presents you with a real-time view of all the connections between your containerized processes, hosts, and cloud instances. Dynatrace includes root cause analysis and the ability to replay problems to see how they evolved over time.
 
 {{% /capture %}}
+
+### New Relic Kubernetes monitoring integration
+
+[New Relic Kubernetes](https://docs.newrelic.com/docs/integrations/host-integrations/host-integrations-list/kubernetes-monitoring-integration) integration provides increased visibility into the performance of your Kubernetes environment in. New Relic's Kubernetes integration instruments the container orchestration layer by reporting metrics from Kubernetes objects. The integration gives you insight into your Kubernetes nodes, namespaces, deployments, replica sets, pods, and containers.
+
+Key capabilities as below:
```
Comment (Bradamant3) [inline content/en/docs/tasks/debug-application-cluster/resource-usage-monitoring.md]: rm "as below"

File: content/en/docs/tasks/debug-application-cluster/resource-usage-monitoring.md [hunk 2]
```diff
@@ -96,3 +96,13 @@ With [Dynatrace Kubernetes monitoring](https://www.dynatrace.com/technologies/cl
...
+View your data in pre-built dashboards for immediate insight into your Kubernetes environment.
+Create your own custom queries and charts in Insights from automatically reported data.
+Create alert conditions on Kubernetes data.
+Learn more on this [page](https://docs.newrelic.com/docs/integrations/host-integrations/host-integrations-list/kubernetes-monitoring-integration)
```
Comment (Bradamant3) [inline content/en/docs/tasks/debug-application-cluster/resource-usage-monitoring.md]: New content at the bottom of the page should be inserted before the closing capture tag for the template shortcode (currently at l98). Please move -- thanks

**Comment by njain1985** [comment]
Implemented the suggested changes, please approve PR.

**Comment by zparnold** [comment]
Also looks like @Bradamant3 has already started the review
/unassign
/assign @Bradamant3

**Comment by Bradamant3** [comment]
@njain1985 apologies for taking so long to merge, and thanks for addressing review comments!

/lgtm
/approve

**Pull request merged**